network/iptables.md

```diff
@@ -43,20 +61,49 @@ $ iptables -I DOCKER-USER -m iprange -i ext_if ! --src-range 192.168.1.1-192.168
 
 You can combine `-s` or `--src-range` with `-d` or `--dst-range` to control both
 the source and destination. For instance, if the Docker daemon listens on both
-192.168.1.99 and 10.1.2.3, you can make rules specific to `10.1.2.3` and leave
+`192.168.1.99` and `10.1.2.3`, you can make rules specific to `10.1.2.3` and leave
 `192.168.1.99` open.
 
 `iptables` is complicated and more complicated rules are out of scope for this
 topic. See the [Netfilter.org HOWTO](https://www.netfilter.org/documentation/HOWTO/NAT-HOWTO.html)
 for a lot more information.
 
+## Docker on a router
+
+Docker also sets the policy for the `FORWARD` chain to `DROP`. If your Docker
+host also acts as a router, this will result in that router not forwarding
+any traffic anymore. If you want your system to continue functioning as a
+router, you can add explicit `ACCEPT` rules to the `DOCKER-USER` chain to
+allow it:
+
+```bash
+$ iptables -I DOCKER-USER -i src_if -o dst_if -j ACCEPT
+```
 
 ## Prevent Docker from manipulating iptables
 
 It is possible to set the `iptables` key to `false` in the Docker engine's configuration file at `/etc/docker/daemon.json`, but this option is not appropriate for most users.  It is not possible to completely prevent Docker from creating `iptables` rules, and creating them after-the-fact is extremely involved and beyond the scope of these instructions. Setting `iptables` to `false` will more than likely break container networking for the Docker engine.
 
 For system integrators who wish to build the Docker runtime into other applications, explore the [`moby` project](https://mobyproject.org/).
 
+## Note on the `--ip` option
+
+By default, the Docker daemon will expose ports on the `0.0.0.0` address, i.e.
+any address on the host. If you want to change that behavior to e.g. only
+expose ports on an internal IP address, you can use the `--ip` option to
+specify a different IP address. This method heightens security (connections
+are not allowed on IPs the service is not bound to), without the necessity
+for disabling the iptables mechanisms altogether, because they do honor this
+setting and do not expose services on IPs they shall not be exposed on.
+
+However, setting `--ip` only changes the _default_,
```

Comment:
I don't think this paragraph adds much value , especially the other container management systems section. 

LGTM on the rest

Reply:
@arkodg thanks for the review! I shortened that section and removed the part in question. I'd like to keep the part that explains that `--ip` only sets a default though, because I tried to use that option to prevent my containers from going public, and it surprised me that this didn't work. What do you think? Is there a section that would fit better, where we could move it to?